charts/vector/templates/_helpers.tpl

```diff
@@ -241,9 +241,95 @@ Print the `url` option for the Vector command.
   {{- if $.Values.customConfig -}}
     {{- if $.Values.customConfig.api -}}
       {{- if $.Values.customConfig.api.address -}}
---url {{ printf "http://%s/graphql" $.Values.customConfig.api.address }}
+{{ print "\\" }}
```

Comment:
Should this line be indented?

Reply:
No, it's getting used at the end of the command [here](https://github.com/vectordotdev/helm-charts/blob/708069a986676afa244736accaea896e9c5b0874/charts/vector/templates/_helpers.tpl#L198), to provide some line-wrapping functionality since appending `--url http://127.0.0.1:8686/graphql` or what not gets pretty far into the right half of the screen.